app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from __future__ import annotations
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (NotVaccinatedError, OutdatedVaccineError):
```

Comment:
NotVaccinatedError and OutdatedVaccineError is child of VaccineError, so you can handle only this exception
```suggestion
        except VaccineError:
```